### DISCUSSION TITLE
Student Developer Pack: My students keep getting denied

### DISCUSSION THREAD

**Discussion by jalee9** [community/community#50818] [GitHub Education]
Greetings! 

I'm an instructor at an institution that utilizes GitHub Global Campus. In the past, our students have always been able to sign up for a student developer pack without issue.  This semester, my students seem to keep getting denied access.  After walking through the process with a few students to make sure it wasn't user error, I could not see anything they were doing wrong. 

Students at our institution have a slightly different email domain.  I'm not sure if that's causing the issue (it was not an issue in the past). Does anyone have suggestions for getting students approved for the developer pack? 

Thanks in advance.

Jennifer

**Comment by samus-aran** [comment]
Hi @jalee9 

Appreciate you taking the time to assist your students on applying for GitHub Global Campus. Unfortunately we can not comment on individual applications without one of your students reaching out. However if you can identify that their domain is different than what your selected school indicates then we ask users to raise a [support ticket](https://support.github.com/contact/education) regarding to incorrect information of their school to be updated.

Hope this helps! Let us know how you get along 👍
